apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
> About the naming for API wrapper/shim, I'm fine for the both name, but just wondering why not calling service in general. Because it would be run in worker eventually.

The goal is to further separate the layers by naming. So that it's easier to know what we're doing in the code.

> how could we define the role of activity_handler in Oleg's activity service? 

activity_handler lives in the view; it's the "shim" "client" ;) (not everything is clear in my head either)